src/pbt/client/airflow/mwaa.py

```diff
@@ -6,29 +6,31 @@
 
 import boto3
 import requests
-from tenacity import retry_if_exception_type, stop_after_attempt, wait_fixed, retry
+from tenacity import retry, retry_if_exception_type, stop_after_attempt, wait_fixed
 
 from . import AirflowRestClient
-from ...utils.exceptions import DagNotAvailableException, DagFileDeletionFailedException, DagUploadFailedException, DagListParsingFailedException
+from ...utils.exceptions import (
+    DagFileDeletionFailedException,
+    DagListParsingFailedException,
+    DagNotAvailableException,
+    DagUploadFailedException,
+)
 from ...utils.project_models import DAG
 
 
 class MWAARestClient(AirflowRestClient, ABC):
-
     def __init__(self, environment_name: str, region: str, access_key: str, secret_key: str):
         self.environment_name = environment_name
         self.aws_s3_client = boto3.client(
-            "s3",
-            region_name=region,
-            aws_access_key_id=access_key,
-            aws_secret_access_key=secret_key
+            "s3", region_name=region, aws_access_key_id=access_key, aws_secret_access_key=secret_key
         )
 
-        self.mwaa_client = boto3.client("mwaa", region_name=region, aws_access_key_id=access_key,
-                                        aws_secret_access_key=secret_key)
+        self.mwaa_client = boto3.client(
+            "mwaa", region_name=region, aws_access_key_id=access_key, aws_secret_access_key=secret_key
+        )
 
         self.environment = self.mwaa_client.get_environment(Name=environment_name)
-        self.dag_s3_path = self.environment['Environment']['DagS3Path']
+        self.dag_s3_path = self.environment["Environment"]["DagS3Path"]
```

Comment:
super nit:

aren't both equivalent single quote and double quotes ?

Reply:
black did it :)